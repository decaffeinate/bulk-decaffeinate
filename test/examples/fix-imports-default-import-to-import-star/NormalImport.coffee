NamedExport = require './NamedExport'
console.log NamedExport.value
