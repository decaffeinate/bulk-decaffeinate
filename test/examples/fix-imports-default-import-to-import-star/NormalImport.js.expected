// TODO: This file was created by bulk-decaffeinate.
// Sanity-check the conversion and remove this comment.
import * as NamedExport from './NamedExport';
console.log(NamedExport.value);
